.github/workflows/kernelctf-submission-verification.yaml

```diff
@@ -83,20 +84,34 @@ jobs:
           ref: ${{ env.PR_REF }}
           fetch-depth: 0
 
+      - name: Checkout master for dependencies script
+        if: startsWith(matrix.target, 'android')
+        uses: actions/checkout@v4
+        with:
+          path: master
+          ref: master
+
       - name: List files
         run: find .
 
       - name: Backup original exploit
         run: mv $EXPLOIT_DIR/exploit ./
 
-      - name: Install kernelXDK
+      - name: Install Android dependencies (exploit-build-only mode)
+        if: startsWith(matrix.target, 'android')
+        run: |
+          chmod +x master/kernelctf/server/get_android_dependencies.sh
```

Comment:
We should add a +x to the git repo too, then we don't need this step and other users can also execute this script without having to chmod first.